app-backup/borgbackup/borgbackup-1.2.0.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Deduplicating backup program with compression and authenticated encryption"
+DESCRIPTION="BorgBackup (short: Borg) is a deduplicating backup program.
+Optionally, it supports compression and authenticated encryption.
+
+The main goal of Borg is to provide an efficient and secure way to backup data.
+The data deduplication technique used makes Borg suitable for daily backups
+since only changes are stored.
+The authenticated encryption technique makes it suitable for backups to not
+fully trusted targets."
+HOMEPAGE="https://github.com/borgbackup/borg"
+COPYRIGHT="201-2021 Mathias Vogelgesang"
+LICENSE="MIT"
+REVISION="1"
```

Comment:
the homepage is https://www.borgbackup.org/ .

the year in the (C) looks strange.

what's this copyright and license referring to? as it is written here (homepage + copyright), it seems like it refers to borg. but then it would be wrong. or does it refer to the recipe?

Reply:
Thanks for replying, I'll change the homepage, typo in the copyright year, think I searched through the code to find the copyright dates, which would you prefer?

Reply:
i still don't know what the (c) is referring to. borg or recipe?

Reply:
borg, we don't copyright the recipes

Reply:
Guess I should use the dates in LICENSE? https://github.com/borgbackup/borg/blob/master/LICENSE#L1

Reply:
```
Copyright (C) 2015-2022 The Borg Collective (see AUTHORS file)
```

And license is the BSD license.

Reply:
which clause?

Reply:
BSD 3-clause